app/records/models.py

```diff
@@ -315,13 +314,13 @@ def related_materials(self) -> tuple[dict[str, Any], ...]:
                     format_link(val, inc_msg) for val in item.get("links", ())
                 ),
             )
-            for item in self.template.get("relatedMaterials", ())
+            for item in self.get("relatedMaterials", ())
         )
 
     @cached_property
     def description(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("description", "")
+        return format_extref_links(self.get("description", ""))
```

Comment:
1. Any thoughts of marking only specific record fields with `django.utils.safestring.mark_safe` - to mark HTML in the field safe for output instead of marking safe in the templates - description field for example?
2. Internal link not resolving for http://localhost:65533/catalogue/id/C361/ 
   Ex: Papers of Francis Kingdon-Ward, Explorer and Botanist, in [RM 1](http://localhost:65533/catalogue/id/C361/c5f5-0f02-4d03-9b5c-8ec7973794b7);
   Should be Papers of Francis Kingdon-Ward, Explorer and Botanist, in [RM 1](http://localhost:65533/catalogue/id/c5f5-0f02-4d03-9b5c-8ec7973794b7);

Reply:
It looks like this was broken in `ds-wagtail` anyway. I want to understand more about the formats of these IDs as they are very inconsistent.

Will push up a fix for this in a sec.

Reply:
Here's something in code to know about the various types

ID formats 
https://github.com/nationalarchives/ds-search/blob/bfefa2a41cd98798d3f6dc95e6deaf4fed7a9f60/test/records/test_id_formats.py#L10-L17

Regex
https://github.com/nationalarchives/ds-search/blob/bfefa2a41cd98798d3f6dc95e6deaf4fed7a9f60/app/records/converters.py#L4-L7

Reply:
Updated the code and the tests.

`c5f5-0f02-4d03-9b5c-8ec7973794b7` is not a valid GUID.

`4d8dae2c-b417-4614-8ed8-924b9b4beeac` (which was in the tests) is valid.

Reply:
As mentioned above, Ignore as its a data issue. I think the `format_extref_links` should handle this error.